firefox-ios/firefox-ios-tests/Tests/L10nSnapshotTests/L10nSuite1SnapshotTests.swift

```diff
@@ -172,6 +177,7 @@ class L10nSuite1SnapshotTests: L10nBaseSnapshotTests {
         // Disable due to issue #7521
         // navigator.goto(BookmarksPanelContextMenu)
```

Comment:
This seems resolved Issue [#7521 ](https://github.com/mozilla-mobile/firefox-ios/issues/7521)is from 2020